invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_records_search/index.js

```diff
@@ -15,14 +16,13 @@ import {
 } from "./components";
 import {
   RDMBucketAggregationElement,
+  RDMCountComponent,
   RDMRecordFacets,
   RDMRecordFacetsValues,
-  RDMToggleComponent,
-} from "../../search/components";
-import {
   RDMRecordSearchBarElement,
-  RDMCountComponent,
-} from "../../search/components";
+  RDMToggleComponent,
+} from "../search/components";
+import { RDMRecordSearchBarElement } from "../search/components";
```

Comment:
Here I guess this import is unnecessary right? It is already imported in line 22...